src/protocol/socket.ts

```diff
@@ -113,6 +114,12 @@ function onSocketMessage(evt: MessageEvent<any>) {
 function onSocketClose() {
   setStatus("Disconnected.");
   log("Socket Closed");
+
+  store.dispatch(
```

Comment:
is the store global? Is there any way to pass it in... 

perhaps ... https://gist.github.com/73b610bde300ea6e50c5ef58ad18ac90

Reply:
it is global and it does work when i test so i'm not sure why it's throwing an error on store. but i'll add your changes in and see if it fixes that error